sources/RuleRunner.cpp

```diff
@@ -85,7 +86,7 @@ status_t DeleteAction(const BMessage& action, entry_ref& ref);
 #undef B_TRANSLATION_CONTEXT
 #define B_TRANSLATION_CONTEXT "RuleRunner"
 
-#define PAIR(s)	{ s, B_TRANSLATE(s) }
+#define LOCALIZE(s)	{ s, B_TRANSLATE(s) }
```

Comment:
I wondered why not use B_TRANSLATE directly. I replaced LOCALIZE with B_TRANSLATE, which resulted in:

`/Source/Filer/sources/RuleRunner.cpp:229: uninitialized const member `NamePair::locale'`

So, I guess it's needed, but I'm to bad a coder to know why... :)